### ISSUE TITLE
Getting a ton of requests when a Mastodon server comments on one of my posts

### ISSUE THREAD

**Issue by limdingwen** [pixelfed/pixelfed#3831]
Hi, I tried commenting on one of my Pixelfed posts from Mastodon, with public visibility. That caused a ton of "GET /users/[name]" requests from servers that I recognise (possibly my followers' servers + relay servers) and caused my computer to crash after creating too many Apache instances.

I'm using Docker, and I'm using nginx in Docker as well in the same `docker-compose.yml` file.. I added this into `docker-nginx.conf` and it seemed to have worked:

`proxy_cache_path /var/cache/nginx levels=1:2 keys_zone=CACHE:10m inactive=7d max_size=1g;`

and

```
location / {
....
	        proxy_cache CACHE;
	        proxy_cache_valid 200 7d;
	        proxy_cache_valid 410 24h;
	        proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
	        add_header X-Cached $upstream_cache_status;
}
```

After this, it seems like the servers have cached the response (not sure, not familiar with Nginx) and have not requested again. Edit: Nope. Seems like they just know about my server now, nothing to do with X-Cache.

This code was taken directly from https://github.com/mastodon/mastodon/blob/main/dist/nginx.conf.

I'm unsure what side effects may come out of doing this.

**Comment by limdingwen** [comment]
I'm not sure if this is related; adding this config seems to reduce the number of times I get the "403 forbidden" with a blank screen issue, which usually requires a "Cmd-Shift-R" (force-reload) to resolve.

Edit: No, still getting it.

**Comment by limdingwen** [comment]
I think the issue was twofold:

1. I commented out the "build" portions of the docker-compose.yml file. I realised that there's no such image as `pixelfed` in Docker Hub... so not sure where it was getting that.
2. I stopped using nginx Docker, and just used normal nginx instead (via systemd).

Essentially, just use the docker-compose.yml file provided, and copy `contrib/docker-nginx.conf` to `/etc/nginx/sites-enabled/` (replace `real.domain` inside) and everything should be fine... sorry for the noise.
